app/services/editions/publishing/EditionsPublishing.scala

```diff
@@ -17,24 +17,36 @@ class EditionsPublishing(publishedBucket: EditionsBucket, previewBucket: Edition
     previewBucket.putIssue(previewIssue)
   }
 
-  def publish(issue: EditionsIssue, user: User, now: OffsetDateTime) = {
+  def proof(issue: EditionsIssue, user: User, now: OffsetDateTime) = {
+
+    val action = PublishAction.proof
+
     // Bump the recently published counters
```

Comment:
should this comment be re-worded?

Reply:
Removed - and matching logging added for publish process.